variables.tf

```diff
@@ -286,3 +291,21 @@ variable "maintenance_window" {
     update_track = "stable"
   }
 }
+
+# ----------------------------------------------------------------------------------------------------------------------
+# To enable Query Insights
+# ----------------------------------------------------------------------------------------------------------------------
```

Comment:
The variable name is pretty clear here. No need to populate more comments for it.

In fact, if we want to clarify the purpose of the variable, we can do so in the description of the variable itself - which will also get included into the documentation pages that are generated by terraform. So that's cleaner than adding comments.

Reply:
done